samples/Common.c

```diff
@@ -533,6 +535,7 @@ STATUS createSampleStreamingSession(PSampleConfiguration pSampleConfiguration, P
     ATOMIC_STORE_BOOL(&pSampleStreamingSession->candidateGatheringDone, FALSE);
 
     pSampleStreamingSession->peerConnectionMetrics.peerConnectionStats.peerConnectionStartTime = GETTIME() / HUNDREDS_OF_NANOS_IN_A_MILLISECOND;
+    pSampleConfiguration->enableIceStats = FALSE;
```

Comment:
Not sure how common it would be for users to try samples with enableIceStats TRUE, but if common enough, would be nice to add a comment assuring that this is the place to do so.

Reply:
README has details on how to set it. I usually refrain from linking to code in readme since the line numbers could change and the link could point to something else.

Reply:
Oh I mean to just add a line comment here in the sample code just above this line